### DISCUSSION TITLE
Misconfigured Administration Team in Project

### DISCUSSION THREAD

**Discussion by zisuu** [WeblateOrg/weblate#10482] [Q&A] [answered]
Hello

We have the issue that one of our User misconfigured the Administration Team of their Weblate Project. They removed the Administration Role from the Administration Team and now they cant even change anything themself:

this is how it currently looks on their project:
![image](https://github.com/WeblateOrg/weblate/assets/11134301/b12c0aa8-593a-4330-b0e5-53177a6e7e74)

this is how it should look:
![image](https://github.com/WeblateOrg/weblate/assets/11134301/63b9e6b6-22c9-4008-8468-28673fd4f0d9)

I tried to reconfigure it as Weblate administratior via WebUI, however I cant find an "Administration" role to add to the Team. I also checked the API Docs but could not find anything that would allow me to restore this misconfiguration.

Is there a way to fix this? thanks a lot!

**Comment by nijel** [comment, marked as answer]
Upgrade Weblate. The  release you are using has a bug not showing this role in the UI.

**Reply by zisuu** [reply]
thanks a lot nijel for your fast response. We will upgrade then 👍
